### ISSUE TITLE
Pod component incorrectly adds a title attribute to the containing element

### ISSUE THREAD

**Issue by ianoxley** [Sage/carbon#1709]
When you pass a `title` attribute to the `Pod` component it erroneously adds a `title` attribute to the component's containing element, with the result that the title tooltip appears when you move the mouse anywhere inside the `Pod` component.

For example:

```js
<Pod
  title="Foobar"
>
  ...
</Pod>
```

This produces the following HTML:

```html
<div class="carbon-pod carbon-pod--left" data-component="pod" title="foobar">
  ...
</div>
```

The `title` prop should only be used to provide the `Pod` component with a heading and nothing more.

## Screenshot
![pod_title_tooltip](https://user-images.githubusercontent.com/52329/37213558-511363e4-23aa-11e8-85ca-038fdc0787d8.gif)

**Comment by Euklidian-Space** [comment]
Hey @ianoxley.  I would like to take a crack at this.  The issue seems to be these lines of code in the render method,

```javascript
if (this.titleIsString()) {
      props.title = this.props.title;
}
```

The props object is then spreaded into the containing element giving it a "title" attribute, which triggers the default html behaviour of the on hover tooltip.  I commented out that if statement and it worked.  I also ran the entire test suite to see if it broke anything.  All tests passed.  It is not apparent to me why that if statement is there at all.  Let me know what you think of this.  Oh and also let me know if you think there should be a test case added to ensure the title attribute doesn't get tacked on.  I will then be glad to submit a PR for this issue.

**Comment by ianoxley** [comment]
Hey @Euklidian-Space, please feel free to have a crack at this - thanks.

It looks like you've got a good handle on how to fix it from what you say above. And a test case to check there's no title attribute sounds like a good idea too 👍 

Just comment here if you have any more questions, etc.

**Comment by Euklidian-Space** [comment]
Hi @ianoxley.  I submitted PR #1800 to address this fix

**Comment by ianoxley** [comment]
Closed by #1800
